Fix Twitter bridge images

somini opened the pull request:
Include the missing changes from #1591, rebased to the latest master.

somini committed:
Fix Twitter bridge images

Credit to @kinoushe

See https://github.com/RSS-Bridge/rss-bridge/issues/1562#issuecomment-639393175

somini committed:
Keep old URI structure

Use the username, not the user ID.

somini committed:
Include Videos and "Animated GIF" as twit enclosures

Credit to @kinoushe for digging into the API docs.

https://github.com/RSS-Bridge/rss-bridge/issues/1562#issuecomment-640320688

somini committed:
Calculate the highest bitrate video

Include that on the enclosure.

somini committed:
Appease linter

somini committed:
Appease linter, again

kinoushe commented:
Just tested and it's working well, thanks again!

One thing, I think the `<a href="{$link}"> ... </a>` will need to be removed from around the video, as clicking the video to play also launches the web link.

somini committed:
Remove surrounding link from videos

Add it on a smaller link besides it.

See
https://github.com/RSS-Bridge/rss-bridge/pull/1595#issuecomment-640989208

kinoushe commented:
After some more testing, should we include the video poster image as an additional enclosure for the video tweets?

```
$item['enclosures'][] = $poster;
$item['enclosures'][] = $video;
```

In apps like Reeder this provides an image to use for the image previews displayed in the app's article list.

somini commented:
Maybe after the video itself? The video is the principal encolosure.

somini committed:
Include video poster on the enclosures.

kinoushe commented:
Perfect, I've just checked and the image previews are still created correctly with this enclosures order.

somini commented:
@rss-bridge This is ready to merge now.
